scripts/check-pkg-for-release.js

```diff
@@ -210,9 +209,7 @@ function checkJSDependencyAddedInMDCPackage() {
   const nameCamel = camelCase(CLI_PACKAGE_JSON.name.replace('@material/', ''));
   const src = fs.readFileSync(path.join(process.env.PWD, MASTER_TS_RELATIVE_PATH), 'utf8');
   const ast = recast.parse(src, {
-    parser: {
-      parse: (code) => parser.parse(code, {sourceType: 'module'}),
-    },
+    parser: require('recast/parsers/typescript'),
```

Comment:
Could we move this `require` to the top of the file and reference it below?

```
const typescriptParser = require('recast/parsers/typescript');
...
const ast = recast.parse(src, {parser: typescriptParser});
```

Reply:
Done